docs/UserGuide.adoc

```diff
@@ -263,7 +263,13 @@ Adds new ingredient into inventory. Previously not existing in inventory before.
 Format: `addIngredient in/INGREDIENT_NAME u/INGREDIENT_UNIT [q/INGREDIENT_QUANTITY] [w/INGREDIENT_WARNING_AMOUNT]`
 
 ****
-• If ingredient quantity or warning amount is not specified, they are respectively set to 0.
+• `INGREDIENT_NAME` and `INGREDIENT_UNIT` must contain only alphabets and spaces, and must contain at least 2 alphabets. (i.e, `in/c` is not accepted while `in/ch` is).
```

Comment:
technical note: at least 2 characters
'a', 'b', 'c' are characters
alphabet refers to the full set of symbols (e.g. the set of all a-z is an alphabet)

Reply:
Edited. Thanks for clarifying!